requirements_dev.txt

```diff
@@ -8,7 +8,7 @@ bump2version>=0.5.11
 # glotaran setup dependencies
 asteval==0.9.21
 numpy==1.19.4; sys_platform != "win32"
```

Comment:
This exception can now be eliminated @s-weigand ?

Reply:
I hope so, but better to test it locally (with the bad windows patch state see https://github.com/numpy/numpy/issues/16744#issuecomment-749851863) first. 😉